src/main/java/uk/gov/hmcts/ccd/service/MessagePublisherRunnable.java

```diff
@@ -59,14 +62,35 @@ private void processUnpublishedMessages(Pageable pageable,
         } else if (!processedEntities.isEmpty()) {
             messageQueueCandidateRepository.saveAll(processedEntities);
             log.info(String.format("%s Published %s messages to destination '%s'",
-                logPrefix, processedEntities.size(), publishMessageTask.getDestination()));
+                                   logPrefix, processedEntities.size(), publishMessageTask.getDestination()
+            ));
         }
     }
 
+    private String getPropertyValue(JsonNode data, String propertyId) {
+
+        return data.get(propertyId).asText();
+    }
+
+    private Message setProperties(Message message, JsonNode data) throws JMSException {
```

Comment:
With the latest implementation can probably remove the `throws`.
```suggestion
    private Message setProperties(Message message, JsonNode data) {
```

Reply:
Removing the throws gives unhandled exception error